crates/daphne-worker-test/Cargo.toml

```diff
@@ -20,14 +20,8 @@ default = ["console_error_panic_hook"]
 [dependencies]
 cap.workspace = true
 cfg-if.workspace = true
-# The `console_error_panic_hook` crate provides better debugging of panics by
-# logging them with `console.error`. This is great for development, but requires
-# all the `std::fmt` and `std::panicking` infrastructure, so isn't great for
-# code size when deploying.
```

Comment:
Comment could stay

Reply:
I think it's misleading. It's not like we are avoiding `std::fmt` or `std::panicking` in the rest of the codebase